Remote Ohai with Target Mode (#1480)
chef/ohai: README.md

- General documentation: <https://docs.chef.io/ohai.html>
- Writing Ohai Plugins documentation: <https://docs.chef.io/ohai_custom.html>

## Remote Ohai

> Ohai can be executed on remote systems to collect data on external systems:

Reviewer: Use `target` mode to execute Ohai and collect data on external systems:

```shell
ohai --target ssh://user@198.51.100.23/?key_files=/etc/chef/remote_ohai.pem
```

> This is using Train backends, which include SSH, WinRM, Docker and local execution by default. Third party Train plugins can be used to extend support.

Reviewer: ```suggestion
Targeting uses Train backends, which include SSH, WinRM, Docker and local execution by default. In addition, you can use third-party Train plugins to extend support.
```

The URL and query parameters depend on the used backend.

## Development Environment:

> Ohai's development dependencies should be installed with bundler. Just run `bundle install` in the root of the repo.

Reviewer: ```suggestion
Install Ohai's development dependencies with bundler. Just run `bundle install` in the root of the repository.
```
